admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,52 @@
-.. Update Notifier and Updater App Are Not the Same
-.. ------------------------------------------------
-
-.. ownCloud has two update tools: the ownCloud core update notifier, and the 
-.. Updater app. Figure 1 shows what you see when the Updater app is enabled: 
-.. both 
-.. the core notifier and the Updater app control panel are visible on your 
-.. admin 
-.. page.
-
-.. .. figure:: images/2-updates.png
-..   :alt: Both update mechanisms displayed on Admin page.
-   
-..   *Figure 1: The top yellow banner is the update notifier, and the Updates 
-..   section is the Updater app.*
-   
-.. The core update notifier has only one function, and that is to display a 
-.. notification when a new ownCloud release is available. Then you decide which 
-.. upgrade method to use. When you maintain your ownCloud server via your Linux 
-.. package manager you should ensure that the Updater app is disabled.
-  
 Prerequisites
 -------------
 
-You should always maintain :doc:`regular backups <backup>` and make a fresh 
-backup before every upgrade.
+We strongly recommend that you always maintain :doc:`regular backups <backup>` as well as make a fresh backup before every upgrade.
+We also recommend that you review any installed third-party apps for compatibility with the new ownCloud release. 
+Ensure that they are all disabled before beginning the upgrade.
+After the upgrade is complete re-enable any which are compatible with the new release.
 
-Then review third-party apps, if you have any, for compatibility with the new 
-ownCloud release. Any apps that are not developed by ownCloud show a 3rd party 
-designation. **Install unsupported apps at your own risk**. Then, before the 
-upgrade, all 3rd party apps must be disabled. After the upgrade is complete you 
-may re-enable them.
+.. warning::
+   Install unsupported apps at your own risk.
 
 .. _owncloud.org/install/:
    https://owncloud.org/install/  
 
-Previous ownCloud Releases
---------------------------
+Upgrade Options
+---------------
 
-You'll find previous ownCloud releases in the `ownCloud Server Changelog 
-<https://owncloud.org/changelog/>`_.
+There are three ways to upgrade your ownCloud server:
+
+#. Using the :doc:`Updater App <update>` (Server Edition only). This is recommended for shared hosters and for users who want an easy way to track different release channels. It is *not available* and *not supported* on the Enterprise edition. This is the recommended method.
+#. Using your :doc:`Linux package manager <package_upgrade>` with our official ownCloud repositories. 
+#. :doc:`Manually upgrading <manual_upgrade>` with the latest ownCloud release from `owncloud.org/install/`_. 
+
+.. note::
+   Please remember that in scale-out deployments (where your database is sitting somewehere else, not on the same machine as the webserver) using the package manager is going to install all the related dependencies, unless the `owncloud-enterprise-files` package is being used.
```

Comment:
I would rather say: 

> please consider that in scale-out deployment scenarios is recommended to follow the manual upgrade procedure since using the Linux package manager is going to install all the related ownCloud dependencies as well (unless the `owncloud-enterprise-files` package is being used) which is actually not required.